circle.yml

```diff
@@ -512,6 +512,53 @@ jobs:
       - spectest
       - collect_coverage_data
 
+  bindings-rust:
+    docker:
+      - image: rust:1-buster
+    steps:
+      - checkout
+      - run:
+          name: Update environment
+          command: |
+            echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
+            apt -qq update
+            apt -yq -t buster-backports install cmake --no-install-recommends
+            apt -yq install libclang-dev clang cmake --no-install-recommends
+            rustup component add rustfmt
+            rustup update
+      - run:
+          name: Check formatting
+          command: |
+            rustfmt --version
+            cargo fmt --all -- --check
+      - run:
+          name: Build
+          command: cargo build
+      - run:
+          name: Test
+          command: cargo test
+
+  bindings-rust-asan-combined:
```

Comment:
What's "combined" about this?

Reply:
The name came from evmc, not sure what it had, can remove the combined.